app/tests/test_download.py

```diff
@@ -0,0 +1,124 @@
+import boto3
+from moto import mock_s3
+
+from app.tests.conftest import mock_standard_user, mock_superuser
+from app.tests.factories import (
+    ConsignmentFactory,
+    FileFactory,
+    FileMetadataFactory,
+)
+
+BUCKET = "test-download"
+CONSIGNMENT_REF = "TDR-123-TEST"
+FILE_PATH = "data/content/folder_a/test_file.txt"
+
+
+def test_invalid_id_raises_404(client):
+    """
+    Given a UUID, `invalid_file_id`, not corresponding to the id
+        of a file in the database
+    When a GET request is made to `/download/invalid_file_id`
+    Then a 404 http response is returned
+    """
+    response = client.get("/download/some-id")
+
+    assert response.status_code == 404
+
+
+def mock_record():
+    consignment = ConsignmentFactory(ConsignmentReference=CONSIGNMENT_REF)
+    file = FileFactory(
+        file_consignments=consignment,
+        FileName="test_file.txt",
+        FilePath=FILE_PATH,
+        FileType="file",
+    )
+
+    metadata = {
+        "date_last_modified": "2023-02-25T10:12:47",
+        "closure_type": "Closed",
+        "description": "Test description",
+        "held_by": "Test holder",
+        "legal_status": "Test legal status",
+        "rights_copyright": "Test copyright",
+        "language": "English",
+    }
+
+    [
+        FileMetadataFactory(
+            file_metadata=file,
+            PropertyName=property_name,
+            Value=value,
+        )
+        for property_name, value in metadata.items()
+    ]
+
+    return file
+
+
+@mock_s3
```

Comment:
I think we can probably remove the mock_s3 decorator either from here or from the tests that call this, can you give that a go?